macformula/state/lvstartup.go

```diff
@@ -0,0 +1,281 @@
+package state
```

Comment:
Can we somehow reference/link the lv startup flow chart?

Reply:
Yea good idea. I think it should definitely go in the docs and then we can leave a link here

Reply:
Leaving this as open issue #92